src/SWP/Bundle/WebRendererBundle/EventListener/ExceptionListener.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * This file is part of the Superdesk Web Publisher Web Renderer Bundle.
+ *
+ * Copyright 2016 Sourcefabric z.u. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2016 Sourcefabric z.ú.
+ * @license http://www.superdesk.org/license
+ */
+namespace SWP\Bundle\WebRendererBundle\EventListener;
+
+use SWP\Bundle\WebRendererBundle\Exception\NoThemeException;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Event\GetResponseForExceptionEvent;
+
+class ExceptionListener
```

Comment:
I think it's better practice to create a generic exception event listener, in case we want to handle any other exception in this way. We should probably handle more exceptions in this way.